### ISSUE TITLE
[noetic] Broken 2D Nav Goal and 2D Pose Estimate button (orientation)

### ISSUE THREAD

**Issue by doisyg** [ros-visualization/rviz#1509]
Last deb version:
```
[ INFO] [${node}]: rviz version 1.14.0
[ INFO] [${node}]: compiled against Qt version 5.12.8
[ INFO] [${node}]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [/rviz_1590590368092672263]: Forcing OpenGl version 0.
[ INFO] [/rviz_1590590368092672263]: Stereo is NOT SUPPORTED
[ INFO] [/rviz_1590590368092672263]: OpenGl version: 4.6 (GLSL 4.6).
```
The 2D Nav Goal button seems broken under noetic. No orientation is published on the topic:
```
rostopic echo /move_base_simple/goal
header: 
  seq: 0
  stamp: 
    secs: 1590590378
    nsecs:  90359437
  frame_id: "map"
pose: 
  position: 
    x: 0.4193878173828125
    y: 0.5008087158203125
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.0
    w: 0.0
---

```

Though the angle is displayed in the console:
`Setting goal: Frame:map, Position(0.419, 0.501, 0.000), Orientation(0.000, 0.000, 0.000, 0.000) = Angle: -2.977`

**Comment by doisyg** [comment]
Same issue for the 2D Pose Estimate button:
```
rostopic echo /initialpose
header: 
  seq: 0
  stamp: 
    secs: 1590596724
    nsecs: 931619017
  frame_id: "map"
pose: 
  pose: 
    position: 
      x: -0.12504196166992188
      y: 0.07902240753173828
      z: 0.0
    orientation: 
      x: 0.0
      y: 0.0
      z: 0.0
      w: 0.0
  covariance: [0.25, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.25, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.06853892326654787]
---
```
with:
`[ INFO] [/rviz_1590596700755421369]: Setting pose: -0.125 0.079 -2.567 [frame=map]`

**Comment by doisyg** [comment]
Just submitted a PR to fix it https://github.com/ros-visualization/rviz/pull/1510. The wrong TF2 conversion side was used for creating the quaternion part from theta and the tf2::Quaternion

**Comment by rhaschke** [comment]
Fixed via #1510.

**Comment by CodeFinder2** [comment]
@rhaschke Thanks for fix, affects me as well! Any plans when this will be available in the Ubuntu packages? Kinda critical for me ... :see_no_evil:
